data_model/src/isi.rs

```diff
@@ -81,10 +85,38 @@ impl Instruction {
             Grant(grant_box) => grant_box.len(),
             Revoke(revoke_box) => revoke_box.len(),
             ExecuteTrigger(execute_trigger) => execute_trigger.len(),
+            SetParameter(set_parameter) => set_parameter.len(),
+            NewParameter(new_parameter) => new_parameter.len(),
         }
     }
 }
 
+/// Sized structure for all possible on-chain configuration parameters.
+#[derive(
+    Debug, Display, Clone, PartialEq, Eq, Decode, Encode, Deserialize, Serialize, IntoSchema, Hash,
+)]
+#[display(fmt = "SET `{parameter}`")]
+pub struct SetParameterBox {
```

Comment:
The mechanism is not in place yet, but this is a rudimentary measure meant to distinguish two very separate cases. Parameter creation or deletion should be very limited if allowed at all, maybe only at genesis or upon API version upgrade. The only sane permission for peers would be setting those parameters, and these two otherwise identical ISIs try to reflect that, as was suggested by @appetrosyan some time ago.

Reply:
Ah, I see. I would maybe argue for slightly different approach with this, but since we'll have to return to this anyway, it's probably better to defer the discussion until when we've decided on details of parameter creation/registration.